services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -738,6 +744,23 @@ assertZeroLegalHoldDevices uid = do
       )
       (numdevs == 0)
 
+testNginzLegalHold :: TestM ()
+testNginzLegalHold = do
+  nginz <- view tsNginz
+  (owner, tid) <- createBindingTeam'
+  (c, t) <- withDummyTestServiceForTeam (Brig.userId owner) tid $ \_chan -> do
```

Comment:
Just curious, trying to understand the code: What do you need the `withDummyTestServiceForTeam` for? Wouldn't `(c, t) <- legalHoldLoginViaNginz ...` work? (if not, why not?).

Reply:
in the test before this PR, we called an internal end-point in brig directly to get the credentials.  that circumvented the integrity tests that (presumably) galley does before approving a LH device, including talking to the device and checking that its certificate and URL are valid.  Now that I need to go through nginz so that the cookie is issued by the same host that will get it back further down in the test, these integrity tests are happening, and fail.

i should have probably double-checked this theory, but that's my mental model, based on an error message to that end that i saw at some point today.